ls1intum/Artemis
# `Development`: Remove 'tracker' from websocket path

We do not really track something, so the path name is misleading (it was part of the generated Jhipster template)

### Checklist
#### General




### Steps for Testing
1. Log in to Artemis
2. Check that web socket messages are sent normally (e.g. results for programming exercises)
3. Check the browser console network tab if the web socket messages are sent normally

### Review Progress






#### Code Review
#### Manual Tests
